ftw/theming/resources/scss/globals/helpers.scss

```diff
@@ -74,6 +75,22 @@
   }
 }
 
+@mixin link-color($color-link: $color-link, $color-link-hover: $color-link-hover) {
```

Comment:
Awesome, I was missing this one now and then 👍 💚 
Is there a scoping problem when reusing global variable names as argument names? I once had problems, but I'm not sure if it was the same situation...